DataMapper/main.bal

```diff
@@ -0,0 +1,75 @@
+type Contact record {
+    string email;
+    string phone_number;
+};
+
+type OrderData record {
+    string order_item_id;
+    string product_id;
+    int quantity;
+    string customer_id;
+};
+
+type InventoryData record {
+    string product_id;
+    string name;
+    string description;
+    string category;
+    decimal price;
+    int stock_quantity;
+    string manufacturer;
+    string release_date;
+};
+
+type Customer record {
+    string name;
+    string email;
+    string phone_number;
```

Comment:
```suggestion
    string phoneNumber;
```

Reply:
This and above mentioned records are all generated.